From 44dac4d2cbaadc2e03e825a223fef741e95b33f2 Mon Sep 17 00:00:00 2001
From: Henrik Grimler <henrik@grimler.se>
Date: Sat, 26 Jul 2025 21:33:21 +0200
Subject: [PATCH] ARM: dts: exynos4210-trats (i9100): fix eMMC and sdcard

sdhci0 is non-removable (eMMC) on i9100, and sdhci2 should be
ACTIVE_LOW to match linux kernel.
---
 arch/arm/dts/exynos4210-trats.dts | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm/dts/exynos4210-trats.dts b/arch/arm/dts/exynos4210-trats.dts
index 88e9c0ed2bb0..57fd8ecae3f6 100644
--- a/arch/arm/dts/exynos4210-trats.dts
+++ b/arch/arm/dts/exynos4210-trats.dts
@@ -240,6 +240,7 @@
 };
 
 &sdhci0 {
+	non-removable;
 	bus-width = <8>;
 	pwr-gpios = <&gpk0 2 0>;
 	status = "okay";
@@ -247,6 +248,6 @@
 
 &sdhci2 {
 	bus-width = <4>;
-	cd-gpios = <&gpx3 4 0>;
+	cd-gpios = <&gpx3 4 1>;
 	status = "okay";
 };
-- 
2.50.1

